apps-rendering/src/components/richLink.tsx

```diff
@@ -171,7 +179,12 @@ const RichLink = (props: {
 		: {};
 
 	const attributes = {
-		css: richLinkStyles(format),
+		css: [
+			richLinkStyles(format),
+			(format.design === ArticleDesign.LiveBlog ||
+				format.design === ArticleDesign.DeadBlog) &&
+				liveBlogRichLinkStyles,
+		],
```

Comment:
A possible alternative to this is a common pattern we've used in AR when swapping styles between different `Format`s:

```ts
const styles = (format: Format): SerializedStyles => {
  switch (format.design) {
    case ArticleDesign.LiveBlog:
    case ArticleDesign.DeadBlog:
      return css(richLinkStyles(format), liveBlogRichLinkStyles);
    default:
      return richLinkStyles(format);
  }
}
```

It's relatively simple to extend if we want to drop in extra styles for other `Design`s.

Reply:
Nice - will update :)